qiita_ware/processing_pipeline.py

```diff
@@ -83,6 +83,47 @@ def _get_qiime_minimal_mapping(prep_template, out_dir):
     return output_fps
 
 
+def _get_sample_names_by_run_prefix(prep_template):
+    """Generates a dictionary of run_prefix and sample names
+
+    Parameters
+    ----------
+    prep_template : PrepTemplate
+        The prep template from which we need to generate the minimal mapping
+
+    Returns
+    -------
+    dict
+        The paths to the qiime minimal mapping files
```

Comment:
Shouldn't this be something like `Dict mapping run_prefix to path of the qiime minimal mapping file.`?